test/e2e/specs/blocks/blocks__jetpack-grow.ts

```diff
@@ -24,7 +24,7 @@ const blockFlows: BlockFlow[] = [
 
 // Interacting with the Premium Content toolbar is currently broken on mobile, so only adding for desktop:
 // https://github.com/Automattic/jetpack/issues/22745
-if ( envVariables.VIEWPORT_NAME === 'desktop' ) {
+if ( envVariables.VIEWPORT_NAME === 'desktop' && ! envVariables.TEST_ON_ATOMIC ) {
```

Comment:
**Suggestion (non-blocking)**: I think this merits a comment inline explaining why we're skipping for now 👍